dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/JaegerOtelCollectorTest.java

```diff
@@ -0,0 +1,96 @@
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.testcontainers.containers.Network;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.containers.SimpleLogConsumer;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import io.openliberty.microprofile.telemetry.internal.apps.spanTest.TestResource;
+import io.openliberty.microprofile.telemetry.internal.utils.TestConstants;
+import io.openliberty.microprofile.telemetry.internal.utils.jaeger.JaegerContainer;
+import io.openliberty.microprofile.telemetry.internal.utils.jaeger.JaegerQueryClient;
+import io.openliberty.microprofile.telemetry.internal.utils.otelCollector.OtelCollectorContainer;
+
+/**
+ * Test exporting traces to a Jaeger server with OpenTelemetry Collector
+ */
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+public class JaegerOtelCollectorTest extends JaegerBaseTest {
+
+    public static final int OTLP_GRPC_PORT = 4317;
+
+    @ClassRule
+    public static Network network = Network.newNetwork();
+
+    @ClassRule
+    public static JaegerContainer jaegerContainer = new JaegerContainer()
+                                                                         .withLogConsumer(new SimpleLogConsumer(JaegerBaseTest.class, "jaeger"))
+                                                                         .withNetwork(network)
+                                                                         .withNetworkAliases("jaeger-all-in-one");
+    @ClassRule
+    public static OtelCollectorContainer otelCollectorContainer = new OtelCollectorContainer("jaeger")
+                                                                                                      .withNetwork(network)
+                                                                                                      .withLogConsumer(new SimpleLogConsumer(JaegerBaseTest.class, "otelCol"))
+                                                                                                      .withNetworkAliases("otel-collector-jaeger");
+
+    public static JaegerQueryClient client;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+
+        client = new JaegerQueryClient(jaegerContainer);
+
+        server.addEnvVar(TestConstants.ENV_OTEL_TRACES_EXPORTER, "otlp");
+        server.addEnvVar(TestConstants.ENV_OTEL_EXPORTER_OTLP_ENDPOINT, otelCollectorContainer.getOtlpGrpcUrl());
+        server.addEnvVar(TestConstants.ENV_OTEL_SERVICE_NAME, "Test service");
+        server.addEnvVar(TestConstants.ENV_OTEL_BSP_SCHEDULE_DELAY, "100"); // Wait no more than 100ms to send traces to the server
+        server.addEnvVar(TestConstants.ENV_OTEL_SDK_DISABLED, "false"); //Enable tracing
+        server.addEnvVar(TestConstants.ENV_COLLECTOR_OTLP_ENABLED, "true");
```

Comment:
I don't think this is needed here.

The `COLLECTOR_OTLP_ENABLED` environment variable should only be needed on the Jaeger _docker container_ to configure it to receive spans via OTLP.